libraries/ESP8266WiFi/examples/HTTPSRequest/HTTPSRequest.ino

```diff
@@ -7,6 +7,11 @@
  *  esp8266/Arduino project continuous integration
  *  build.
  *
+ *  Limitations:
+ *    only RSA certificates
+ *    no support of Perfect Forward Secrecy (PFS)
+ *    TLSv1.2 is supported since SDK 2.4.0-rc1
```

Comment:
What is SDK 2.4.0-rc1? AFAIK it has not reached 2.2.0 yet...

Reply:
The arduino esp sdk was suggested: https://github.com/esp8266/Arduino/releases/tag/2.4.0-rc1